### ISSUE TITLE
Accessibility: DetailsList - Chevron button in list header is not read by screen readers

### ISSUE THREAD

**Issue by haonliu** [OfficeDev/office-ui-fabric-react#8158]
### Environment Information 
 
- __Package version(s)__: 6.142.0

### Please provide a reproduction of the bug in a codepen:
 
https://developer.microsoft.com/en-us/fabric#/components/detailslist/grouped
 
#### Actual behavior:
 
The first chevron button in the list header in Automated checks is not read by ATs.
The first chevron icon, on the header is just read as "clickable"
 
#### Expected behavior:
 
The first chevron icon, on the header should be mentioned by AT as

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes)
 
Requested priority: (High)

**Comment by KevinTCoughlin** [comment]
@haonliu, thanks for the report. Can you please provide the accessibility validation error associated with this issue so we can better triage and address?

**Comment by haonliu** [comment]
Hi  @KevinTCoughlin. Thanks for your quick response. Actually there is no accessibility violations on the component, but it is confusing for AT users when the chevron is read out as 'clickable' without any context. It would be nice if it could have the same behavior as the chevrons in the group headers. i.e. having aria-label and aria-expanded attributes (as shown below) that helps AT users understand the function of the button. 
![image](https://user-images.githubusercontent.com/15974344/53607894-7c0dc380-3b75-11e9-9d56-78d59bbbbf0e.png)

**Comment by KevinTCoughlin** [comment]
@haonliu the PR including the above requeste functionality should be merged today. For usage of the new prop (its opt-in), please see: https://github.com/OfficeDev/office-ui-fabric-react/pull/8253#issuecomment-472575253.

Thanks for your patience!
